pkg/webhook/pod/validating/pod_create_update_handler.go

```diff
@@ -52,6 +52,7 @@ func (h *PodCreateHandler) validatingPodFn(ctx context.Context, req admission.Re
 			allowed, reason, err = h.podUnavailableBudgetValidatingPod(ctx, req)
 		}
 	case admissionv1beta1.Delete, admissionv1beta1.Create:
+		allowed, reason, err = h.workloadSpreadValidatingPod(ctx, req)
```

Comment:
Should return `err` if it is not nil, otherwise it may be overwrote by `podUnavailableBudgetValidatingPod `.